source/basic.d.ts

```diff
@@ -20,13 +20,20 @@ Matches a JSON array.
 
 @category Basic
 */
-export interface JsonArray extends Array<JsonValue> {}
-// TODO: Make it this when targeting TypeScript 4.1:
-// export type JsonArray = JsonValue[];
+export type JsonArray = JsonValue[];
+
+/**
+Matches any valid JSON primitive value.
+
+@category Basic
+*/
+export type JsonPrimitive = string | number | boolean | null;
 
 /**
 Matches any valid JSON value.
 
+@remarks Also see ./Jsonify
```

Comment:
Use the `@see` keyword (make sure you use it correctly)

Reply:
6c6d324f787855560d67590d48c33befdd64efb4